src/widgets/custom_text_dialog.rs

```diff
@@ -22,6 +22,7 @@ use adw::subclass::prelude::*;
 use glib::subclass::Signal;
 use gtk::{gio, glib};
 use std::cell::{Cell, RefCell};
+use std::fs::read_to_string;
 use std::sync::OnceLock;
```

Comment:
Import the gettext function for translation purposes

```suggestion
use std::sync::OnceLock;
use gettextrs::gettext;
```